common/api/geometry-core.api.md

```diff
@@ -183,6 +183,9 @@ export type AnnounceNumberNumberCurvePrimitive = (a0: number, a1: number, cp: Cu
 // @public
 export type AnyCurve = CurvePrimitive | CurveCollection;
 
+// @public
+export type AnyCurvePrimitive = Arc3d | LineSegment3d | LineString3d | BSplineCurve3d | BezierCurve3d | DirectSpiral3d | IntegratedSpiral3d | CurveChainWithDistanceIndex;

```

Comment:
CurveChainWithDistanceIndex is a CurveChain already, not sure what tryAddPath does...I probably would leave this out?

Reply:
No, it is a subclass of CurvePrimitive that has a CurveChain as a member.
Path.tryAddChild will append any it.

Reply:
Ok, wonder if anyone will ever use this. :)